### ISSUE TITLE
Error getting 'loop0' information: Failed to get status of the device loop0: No such device or address (g-bd-loop-error-quark, 1)

### ISSUE THREAD

**Issue by mbiebl** [storaged-project/udisks#1150]
Version: 2.10.0

Since the upgrade of udisks to version 2.10.0, I see a lot of those messages in the journal:
```
Jul 21 08:36:47 pluto systemd[1]: Starting udisks2.service - Disk Manager...
Jul 21 08:36:47 pluto udisksd[756]: udisks daemon version 2.10.0 starting
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop0' information: Failed to get status of the device loop0: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop1' information: Failed to get status of the device loop1: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop2' information: Failed to get status of the device loop2: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop3' information: Failed to get status of the device loop3: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop4' information: Failed to get status of the device loop4: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop5' information: Failed to get status of the device loop5: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop6' information: Failed to get status of the device loop6: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop7' information: Failed to get status of the device loop7: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop0' information: Failed to get status of the device loop0: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop1' information: Failed to get status of the device loop1: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop2' information: Failed to get status of the device loop2: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop3' information: Failed to get status of the device loop3: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop4' information: Failed to get status of the device loop4: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop5' information: Failed to get status of the device loop5: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop6' information: Failed to get status of the device loop6: No such device or address (g-bd-loop-error-quark, 1)
Jul 21 08:36:47 pluto udisksd[756]: Error getting 'loop7' information: Failed to get status of the device loop7: No such device or address (g-bd-loop-error-quark, 1)
```

**Comment by tbzatek** [comment]
This should be fixed by #1148 and https://github.com/storaged-project/libblockdev/pull/941

**Comment by mbiebl** [comment]
Thanks, I pulled the patch into the Debian package and the error messages are gone.
